src/lib/orionld/serviceRoutines/orionldPostEntities.cpp

```diff
@@ -62,149 +62,7 @@ extern "C"
 #include "orionld/context/orionldUserContextKeyValuesCheck.h"    // orionldUserContextKeyValuesCheck
 #include "orionld/context/orionldUriExpand.h"                    // orionldUriExpand
 #include "orionld/serviceRoutines/orionldPostEntities.h"         // Own interface
-
-
-
-// -----------------------------------------------------------------------------
-//
-// orionldValidName -
-//
-bool orionldValidName(char* name, char** detailsPP)
```

Comment:
Here is the function to validate an attribute name.
Its name should change perhaps ... `orionldValidAttributeName`